tools/dm-task-configuration-file-intro.md

```diff
@@ -0,0 +1,183 @@
+                                                    # SQL file based on the directory)
+
+syncers:                                            # Configuration arguments of running Syncer
+    global:
+        worker-count: 16                            # The number of threads that synchronize binlog events concurrently in Syncer
+        batch: 1000                                 # The number of SQL statements in a transaction batch that Syncer 
+                                                    # synchronizes to the downstream database
+        max-retry: 100                              # The retry times of the transactions with an error that Syncer synchronizes
+                                                    # to the downstream database (only for DML operations)
+```
+
+References:
+
+- For details of `user-filter-1`, see [Filter rules of binlog events](../tools/dm-task-config-argument-description.md#filter-rules-of-binlog-events).
+- For details of `instance`, see [Black white list rule](../tools/dm-task-config-argument-description.md#black-white-list-rule).
+- For details of `instance-1` of `column-mappings`, see [Column mapping rule](../tools/dm-task-config-argument-description.md#column-mapping-rule).
+
+## Instance configuration
+
+This part defines the subtask of data synchronization. DM supports synchronizing data from one or multiple MySQL instances to the same instance.
+
+```
+mysql-instances:
+    -
+        config:                                    # The upstream database configuration corresponding to `instance-id`
+            host: "192.168.199.118"
+            port: 4306
+            user: "root"
+            password: "1234"                       # Requires the password encrypted by dmctl
+        instance-id: "instance118-4306"            # The MySQL instance ID. It corresponds to the upstream MySQL instance. It is 
+                                                   # not allowed to set it to an ID of a MySQL instance that is not within the 
+                                                   # DM-master cluster topology.
+
+        meta:                                      # The position where the binlog synchronization starts when the checkpoint of 
+                                                   # the downstream database does not exist. If the checkpoint exits, this 
+                                                   # configuration does not work. 
+            binlog-name: binlog-00001
+            binlog-pos: 4
+
+        route-rules: ["user-route-rules-schema", "user-route-rules"]       # Routing rules selected from `routes` above
+        filter-rules: ["user-filter-1", "user-filter-2"]                   # Filter rules selected from `filters` above
+        column-mapping-rules: ["instance-1"]                               # The column mapping rule name of the matched table of 
```

Comment:
this comment can also be:
Column mapping rules selected from `column-mappings` above